packages/salesforcedx-vscode-core/test/vscode-integration/commands/forceSourceRetrieveMetadata/forceSourceRetrieveCmp.test.ts

```diff
@@ -71,11 +65,18 @@ describe('Force Source Retrieve and open', () => {
           "type": "ApexClass",
           "filePath": "force-app/main/default/classes/TestClass.cls"}
   ] }}`;
-  const getCmdResultStub: SinonStub = sb
-    .stub(CommandOutput.prototype, 'getCmdResult')
-    .returns(resultData);
+  let getCmdResultStub: SinonStub;
 
-  afterEach(async () => {
+  beforeEach(() => {
+    sb = createSandbox();
```

Comment:
@LisandroFernandezSF I think the caveat is sandbox and stubs should be created **inside** beforeEach.

Reply:
I would be surprised if this was the issue, I've created sandboxes outside of these contexts without a problem. Sandboxes only need to be created once.

Reply:
Oh but the actual stubbing i think you're right 👍

Reply:
yeah, I think the bigger issue here is why core integration test failures are not surfaced in Circle CI. I created W-8190289 for it. @brpowell I remember you also mentioned false positive in core tests not long ago.

Reply:
@brpowell @xyc Do you want me to move the `let sb: SinonSandbox;` inside the beforeEach function?

Reply:
@xyc yeah that is alarming especially in this case when there were very clearly test failures. In the case that I experienced the test runner didn't even initialize so that made a little more sense. @LisandroFernandezSF not essential just a tip that it can be initialized once, up to you.